src-docs/src/views/guidelines/writing.js

```diff
@@ -233,6 +233,19 @@ export default {
               </GuideRuleWriting>
             </GuideRuleExample>
           </GuideRule>
+
+          <GuideRule heading="" description="">
+            <GuideRuleExample
+              type="do"
+              text="Do. Avoid unneeded words in button labels.">
```

Comment:
The `Do`s and `Don't`s already get applied via the `type`.

```suggestion
              text="Avoid unneeded words in button labels.">
```